config/auto-doc.php

```diff
@@ -0,0 +1,184 @@
+<?php
```

Comment:
no need to add the config file to the package, to test the correct behavior of the config updating - please mock file_get_contents which will return the config content as you wish